libs/platform/src/lib/components/thumbnail/thumbnail.component.html

```diff
@@ -0,0 +1,25 @@
+<div [ngClass]="isHorizontal ? 'fdp-thumbnail-container--horizontal' : 'fdp-thumbnail-container'">
+    <fdp-thumbnail-image
+        *ngIf="!isHorizontal"
+        [mediaList]="mediaList"
+        [isHorizontal]="isHorizontal"
+        (thumbNailClicked)="thumbNailClickHandle($event)"
+    ></fdp-thumbnail-image>
+    <div class="fdp-media-container" [ngClass]="{ 'fdp-media-container--horizontal': 'isHorizontal' }">
```

Comment:
I don't think you need the `[ngClass]` and the `.fdp-media-container--horizontal` class definition here. Instead take advantage of the parent class definition and modify your CSS rule from this:

```scss
.fdp-media-container--horizontal {
  margin-bottom: 4rem;
}
```

to this:

```scss
.fdp-thumbnail-container--horizontal .fdp-media-container {
  margin-bottom: 4rem;
}
```

In this way, your taking advantage of the CSS logic, and do not need to add the extra JavaScript.